### PR TITLE
PT-2162: Make location of countries.json configurable

### PR DESCRIPTION
**Pull request by eMazeika**
https://virtocommerce.atlassian.net/browse/PT-2159

### CODE REVIEW

**Review by Vectorfield4 — APPROVED** [review]

**Comment by VitalyTartynov** [comment]
@Vectorfield4 I'm not sure about this implementation. Wait confirmation from @artem-dudarev

**Comment by artem-dudarev** [comment]
Just move this file to some other directory (root or wwwroot).
This file is not generated at runtime and it doesn't contain any secret information.

**Comment by artem-dudarev** [comment]
It's just a static list of countries - why would anyone customize its location?

File: src/VirtoCommerce.Platform.Core/PlatformOptions.cs [hunk 1]
```diff
@@ -27,7 +27,10 @@ public class PlatformOptions
         //Local path to private key for signing license
         public string LicensePrivateKeyPath { get; set; }
 
-        //Url for discovery sample data for initial installation
+        //Local path for countries list
+        public string CountriesFilePath { get; set; } = "localization/common/countries.json";
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Core/PlatformOptions.cs:31]: I mean this [path](https://github.com/VirtoCommerce/vc-platform/tree/master/src/VirtoCommerce.Platform.Web/wwwroot/Localizations). 
```suggestion
        public string CountriesFilePath { get; set; } = "wwwroot/localizations/common/countries.json";
```

**Review by Vectorfield4 — APPROVED** [review]

File: src/VirtoCommerce.Platform.Core/PlatformOptions.cs (hunk 1 above)
Comment (eMazeika) [reply src/VirtoCommerce.Platform.Core/PlatformOptions.cs:31]: wwwroot/Localizations is dedicated to actual localizations. Can @tatarincev suggest something else?

**Review by tatarincev — APPROVED** [review]

**Pull request merged**